contracts/strategies/matic/multi/StrategyAaveMaiBal.sol

```diff
@@ -17,7 +17,7 @@ import "../../../base/SlotsLib.sol";
 /// @title AAVE->MAI->BAL Multi Strategy Polygon Implementation
 /// @author belbix, bogdoslav
 contract StrategyAaveMaiBal is AaveMaiBalStrategyBase {
-  using SlotsLib for bytes32;
+  using SlotsLib for uint;
```

Comment:
you can remove it here

Reply:
Remove what? SlotsLib I need to set _MAI_CAM_PIPE_SLOT  & _MAI_STABLECOIN_PIPE_SLOT

Reply:
I going to remove it